actions/hosted_customer.tsx

```diff
@@ -179,3 +180,19 @@ export function retryFailedHostedCustomerFetches() {
         return {data: true};
     };
 }
+
+export function getTrueUpReviewBundle(): ActionFunc {
```

Comment:
This is a post verb, but the name indicates a get operation. Maybe `submitTrueUpReview`?